src/components/adaptive-cards/AdaptiveCard/AdaptiveCard.jsx

```diff
@@ -4,6 +4,7 @@ import {Template} from 'adaptivecards-templating';
 import webexComponentClasses from '../../helpers';
 
 import Component, {acPropTypes, registerComponent} from '../Component/Component';
+import '../Action/ActionOpenURL/ActionOpenUrl';
```

Comment:
let's keep the structure flat and put Action.OpenUrl in `src/components/adaptive-cards/ActionOpenUrl/`